### PR TITLE
GitHub Example plots and some standardization

### PR DESCRIPTION
**Pull request by cansavvy**
# Description

This is adding plots and info for the github data examples. Related to #6 

I added plots for the github stuff but also adding more repos that we can pull from. 

I'm also adding here some standard structure items for each page. I think each page should have a structure like this: 

1) Preview - snazzy plot that shows a preview of what the data could look like 
2) Data Information - Table that shows what data are available 
3) Setting up - How we actually go about setting it up
4) Customization -- what if the data you want isn't there? How to customize

### CODE REVIEW

File: citations.Rmd [hunk 1]
```diff
@@ -39,6 +29,18 @@ citations %>%
   count(original_paper)
 ```
 
+## Data information 
+
+Column information:
```
Comment (cansavvy) [inline citations.Rmd:38]: I also reordered the citation data

File: calendly.Rmd [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ---
 title: "Calendly"
-output: html_document
+output:
```
Comment (cansavvy) [inline calendly.Rmd:3]: Added TOCs to each page

**Comment by kweav** [comment]
I'm only seeing metricminer.org and metricminer repos in the preview plots, and I thought from the description there would be OTTR repos and no more metricminer.org. Where should I be looking?

**Comment by cansavvy** [comment]
> I'm only seeing metricminer.org and metricminer repos in the preview plots, and I thought from the description there would be OTTR repos and no more metricminer.org. Where should I be looking?

The additional data won't show up before we end up merging this to main

**Comment by kweav** [comment]
I like the general concept for standard structure. I'm a little confused by the setup section. Is the setup part referring to setting up metric collection, and if so, is that necessary as you can point them to documentation where you have that previously? Or is the setup part not setting up the metric collection and instead setting up plots from the data?

**Comment by kweav** [comment]
From looking at the produced file, I think it's metric collection. And it is nice to have the specific information there. So then I guess it's a question of where does adding setting up more plots go with the general structure?

**Comment by kweav** [comment]
Perhaps make the code for producing the plots a collapsible section that can be viewed if wanted?

**Comment by cansavvy** [comment]
When you are talking about "adding setting up more plots"

Are you talking about adding a section that describes how someone can adjust the ggplots and add more plots?

Probably makes sense to shuttle them to ggplot2 tutorials so we don't have to remake the wheel. And then we can put that underneath "customization" sections?

What do you think?

**Comment by kweav** [comment]
I didn't mean customizing a plot; I was referring to how there are two plots there in the preview section and on my first reading of the proposed structure, I understood it to mean show a pretty plot to get their attention. From looking at the file, I know there are multiple plots there, so maybe I just misunderstood the goal of putting the plots first.

**Comment by kweav** [comment]
I do like the idea of adding a link to ggplot documentation in the customization section. So customization of the data collected and customization of plots. That makes sense to me

**Comment by cansavvy** [comment]
> I understood it to mean show a pretty plot to get their attention. From looking at the file, I know there are multiple plots there, so maybe I just misunderstood the goal of putting the plots first. 


I still see this as the goal, 👍 so I'm confused on where the misunderstanding is. What's the discrepancy you are seeing versus what you expected?

**Comment by kweav** [comment]
> Preview - snazzy plot that shows a preview of what the data could look like

From the description, I thought it was just a single snazzy plot to get people's attention/show an example of how the collected data could be displayed

For something like google analytics, there could be half a dozen plots easily. And for the GitHub one there are two there right now. I think with the collapsible code to produce the plots section that helps keep it in line with the preview idea, but I don't expect that several plots stay in line with a brief, attention getting preview.

**Comment by cansavvy** [comment]
> > Preview - snazzy plot that shows a preview of what the data could look like
> 
> From the description, I thought it was just a single snazzy plot to get people's attention/show an example of how the collected data could be displayed
> 
> For something like google analytics, there could be half a dozen plots easily. And for the GitHub one there are two there right now. I think with the collapsible code to produce the plots section that helps keep it in line with the preview idea, but I don't expect that several plots stay in line with a brief, attention getting preview.

I like this plan! Let's do this! So collapse the code but keep the plots shown or collapse both?

**Comment by kweav** [comment]
Keep at least one plot shown. Probably ok to keep all the plots shown as long as they have titles/short captions

**Comment by cansavvy** [comment]
> Keep at least one plot shown. Probably ok to keep all the plots shown as long as they have titles/short captions

I like it!

**Pull request merged**